### ISSUE TITLE
2.2 RC Installation Fails - EE 2.2.0-RC1.2 / B2B 1.1.0-RC1.2 - Key column 'parent_product_id' doesn't exist in table

### ISSUE THREAD

**Issue by careys7** [magento/magento2#10160]
### Preconditions

1. MySQL 5.7.18
2. PHP 7.0.20
3. Community 2.2.0-RC1.1
4. Enterprise 2.2.0-RC1.2
5. B2B 1.1.0-RC1.2

### Steps to reproduce

1. Run installation using CLI, eg:

```bash
bin/magento setup:install --db-host mysql --db-name magento --db-user root --db-password magento123 --admin-user admin --admin-password magento123 --admin-firstname User --admin-lastname Name --admin-email user@example.com
```

### Expected result

1. Installation Completes

### Actual result

1. Error output:

```
---
[Progress: 207 / 849]
Module 'Magento_BundleStaging':
Installing schema... Upgrading schema...

  [PDOException]
  SQLSTATE[42000]: Syntax error or access violation: 1072 Key column 'parent_product_id' doesn't exist in table
---
```

**Comment by TomashKhamlai** [comment]
Hello, thank you for your submission. I can not reproduce this issue. I will give you details of my Magento installation process and some other details. It is necessary to detect what was the reason of this error.

### Preconditions:
1. `cat /proc/version`
> Output:
```
Linux version 3.10.0-514.el7.x86_64 (builder@kbuilder.dev.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-11) (GCC) ) #1 SMP Tue Nov 22 16:42:41 UTC 2016
```
2. `cat /etc/centos-release`
> Output:
```
CentOS Linux release 7.3.1611 (Core)
```
3. `yum list installed mysql-* > installedMySQL.txt && yum list installed php70* > installedPHP.txt && cat installed*`
> Output:
```
Loaded plugins: fastestmirror, langpacks, post-transaction-actions, replace,
              : tmprepo, versionlock
Installed Packages
mysql-community-client.x86_64             5.7.18-1.el7        @mysql57-community
mysql-community-common.x86_64             5.7.18-1.el7        @mysql57-community
mysql-community-libs.x86_64               5.7.18-1.el7        @mysql57-community
mysql-community-libs-compat.x86_64        5.7.18-1.el7        @mysql57-community
mysql-community-server.x86_64             5.7.18-1.el7        @mysql57-community
Loaded plugins: fastestmirror, langpacks, post-transaction-actions, replace,
              : tmprepo, versionlock
Installed Packages
php70w.x86_64                            7.0.20-1.w7                   @webtatic
php70w-bcmath.x86_64                     7.0.20-1.w7                   @webtatic
php70w-cli.x86_64                        7.0.20-1.w7                   @webtatic
php70w-common.x86_64                     7.0.20-1.w7                   @webtatic
php70w-gd.x86_64                         7.0.20-1.w7                   @webtatic
php70w-intl.x86_64                       7.0.20-1.w7                   @webtatic
php70w-mbstring.x86_64                   7.0.20-1.w7                   @webtatic
php70w-mcrypt.x86_64                     7.0.20-1.w7                   @webtatic
php70w-mysql.x86_64                      7.0.20-1.w7                   @webtatic
php70w-opcache.x86_64                    7.0.20-1.w7                   @webtatic
php70w-pdo.x86_64                        7.0.20-1.w7                   @webtatic
php70w-pear.noarch                       1:1.10.4-1.w7                 @webtatic
php70w-pecl-xdebug.x86_64                2.5.3-1.w7                    @webtatic
php70w-process.x86_64                    7.0.20-1.w7                   @webtatic
php70w-soap.x86_64                       7.0.20-1.w7                   @webtatic
php70w-xml.x86_64                        7.0.20-1.w7                   @webtatic
```
### Steps to reproduse:
1. Clone Magento2 from Github repo:
`cd /var/www/html && git clone https://github.com/magento/magento2.git magento2ce && git checkout tags/2.2.0-RC1.1 && composer install`
2. Create `/etc/httpd/conf.d/magento.vg.conf` and paste the following content:
```
<VirtualHost *:80>
    DocumentRoot "/var/www/html/magento2ce"
    ServerName magento.vg
    <directory "/var/www/html/magento2ce">
        Options Indexes FollowSymLinks
        AllowOverride all
        Require local
        Require all granted
    </directory>
</VirtualHost>
```
3. Make sure that `127.0.0.1       magento.vg` is present in your `/etc/hosts` file, and that there is no conflicting records there.
4. Execute the following code just for no worries about filesystem permissions:
`cd /var/www/html/magento2ce && sudo find var vendor pub/static pub/media app/etc -type f -exec chmod g+w {} -v \; && sudo find var vendor pub/static pub/media app/etc -type d -exec chmod g+ws {} -v \; && sudo chown -Rv :apache . && sudo chmod u+x -v bin/magento`
Look [here](http://devdocs.magento.com/guides/v2.0/install-gde/prereq/file-system-perms.html#perms-set-two-users) for refferrence.
5. Restart httpd.service `sudo systemctl restart httpd`
6. Drop database "magento" if it was created. Backup it if necessary. Create database "magento".
7. Execute the folloving:
```
php bin/magento setup:install --base-url=http://magento.vg/ \
--db-host=localhost --db-name=magento --db-user=root --backend-frontname=admin \
--admin-firstname=Magento --admin-lastname=User --admin-email=user@example.com \
--admin-user=admin --admin-password=123123q --language=en_US \
--currency=USD --timezone=America/Chicago --use-rewrites=1
```
I have taken it from [devdocs article](http://devdocs.magento.com/guides/v2.0/install-gde/install/cli/install-cli-install.html#instgde-install-cli-magento)

I am asking you to try command line installation one more time. If it fails then please provide detailed information about your software environment as did it for you.

**Comment by careys7** [comment]
Hi @TomashKhamlai - it looks like you haven't installed Enterprise / B2B which is where I believe this error is coming from?

**Comment by careys7** [comment]
This is resolved in RC 1.3, thanks for your help
